scripts/getProposalCounts.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  getAccountTypes,
+  Governance,
+  Proposal,
+  ProposalState,
+} from '@models/accounts'
+import { encode } from 'bs58'
+
+const rpc = async (body) => {
+  const res = await fetch('https://api.mainnet-beta.solana.com', {
+    body: typeof body === 'string' ? body : JSON.stringify(body),
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+    },
+  })
+
+  const json = await res.json()
+
+  return json
+}
+
+const getGovernanceAccounts = async (realms) => {
```

Comment:
Can we use `getGovernanceAccounts()` here? https://github.com/solana-labs/governance-ui/blob/0eb810ca07bd90dd672fd47daa192ab9e91c094b/models/api.ts#L98

Reply:
yes we can, and I am keen to re-use code 👍  the only downside of using it in its current form is that it would start making _a lot_ of individual `getProgramAccount` requests instead of a single batched request. This is ok for mango.rpcpool.com but anyone who forks this repo could have rate-limit issues with different RPC providers.

To solve this I was thinking I might refactor getGovernanceAccounts (and now getBorshProgramAccounts) to have the ability to make a batched request like I did in this refactoring https://github.com/johnrees/governance-ui/blob/optimise-scripts/scripts/api.ts#L26-L100

Reply:
this issue of loading a few batched vs lots of non-batched requests is almost solved on one of my local branches

https://github.com/johnrees/governance-ui/blob/request-batching-service-worker/public/sidekick.js

it will automatically batch `getGovernanceAccounts` and any other type of RPC request without needing to make any changes to the current code

I'll open a PR next week, but feel free to reject if not interested

<img width="1321" alt="Screenshot 2021-12-26 at 6 56 59 PM" src="https://user-images.githubusercontent.com/601961/147417500-677c8fea-0f22-4ab1-a9a2-777b5cfb303c.png">

Reply:
This is super cool